code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -362,6 +363,9 @@
 	else
 		return ..()
 
+/obj/item/gun/projectile/revolver/doublebarrel/sawoff(mob/user)
+    . = ..()
+    weapon_weight = WEAPON_MEDIUM
```

Comment:
need a new line.

Reply:
I believe I've fixed it- do correct me if I'm wrong though